_source/_posts/2018-06-05-authentication-vanilla-js.md

```diff
@@ -771,20 +783,27 @@ class App {
     this.signIn = new OktaSignIn({
       baseUrl: 'https://{yourOktaDomain}',
       clientId: '{clientId}',
-      redirectUri: window.location.origin,
+      redirectUri: 'http://localhost:8080',
```

Comment:
Any particular reason you changed this value? `window.location.origin` should work and it's more dynamic.